test/tests/functional/pbs_cray_vnode_per_numa.py

```diff
@@ -159,17 +160,27 @@ def test_settings(self):
 
         # Compare the pbsnodes output when vnode_per_numa_node was unset
         # versus when vnode_per_numa_node was set to False.
-
-        if (len(vnodes_combined) == len(vnodes_combined1)):
+        # List of resources to be ignored while comparing.
+        ignr_rsc = ['license', 'last_state_change_time']
+        len_vnodes_combined1 = len(vnodes_combined1)
+        len_vnodes_combined = len(vnodes_combined)
+        if len_vnodes_combined == len_vnodes_combined1:
             self.logger.info(
                 "pbsnodes outputs are equal in length.")
-            for n in vnodes_combined:
-                if n not in vnodes_combined1:
-                    self.logger.error(
-                        "ERROR vnode %s has differing element." % n['id'])
-                    self.assertTrue(False)
-            self.logger.info(
-                "pbsnodes outputs are the same.")
+            for n in range(0, len_vnodes_combined - 1):
```

Comment:
No need to start your range at 0.  It starts at 0 by default.  You can just say range(len_vnodes_combined - 1)